src/library/scala/math/package.scala

```diff
@@ -219,6 +222,37 @@ package object math {
   /** @group signum */
   def signum(x: Double): Double = java.lang.Math.signum(x)
 
+
+  def floorDiv(x: Int, y: Int): Int = java.lang.Math.floorDiv(x, y)
+
+
+  def floorDiv(x: Long, y: Long): Long = java.lang.Math.floorDiv(x, y)
+
+  def floorMod(x: Int, y: Int): Int = java.lang.Math.floorMod(x, y)
+
+  def floorMod(x: Long, y: Long): Long = java.lang.Math.floorMod(x, y)
+
+  def copySign(magnitude: Double, sign: Double): Double = java.lang.Math.copySign(magnitude, sign)
+
+  def copySign(magnitude: Float, sign: Float): Float = java.lang.Math.copySign(magnitude, sign)
+
+  def nextAfter(start: Double, direction: Double): Double = java.lang.Math.nextAfter(start, direction)
+
+  def nextAfter(start: Float, direction: Float): Float = java.lang.Math.nextAfter(start, direction)
```

Comment:
Why is the signature different to the Java version?

https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#nextAfter-float-double-

The Java version is `nextAfter(float start, double direction)` i.e. a deviation from the float/float or double/double pattern seen elsewhere. Find out why Java is different in this case?